index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jauhari Alafi</title>
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- CSS -->
+    <link rel="stylesheet" href="styles/global.css" />
+    <link rel="stylesheet" href="styles/template.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <a href="#" class="nav-logo">Welcome</a>
+        <a href="#" class="nav-toggle"
+          ><img src="images/icons/hamburger-menu.svg" alt="Menu"
+        /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="headline-primary">
+        <h2>
+          <span class="dark-blue">Hey there. I'm Mariane</span>
+          <br />
+          <span class="green">I'm a software developer</span>
+        </h2>
+      </div>
+      <div class="headline-sub">
+        <p class="dark-blue">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+      </div>
+      <div class="headline-socials">
+        <a href="#" class="social-icon"
+          ><img src="images/icons/twitter.svg" alt="Twitter"
+        /></a>
+        <a href="#" class="social-icon"
+          ><img src="images/icons/linkedin.svg" alt="LinkedIn"
+        /></a>
+        <a href="#" class="social-icon"
+          ><img src="images/icons/medium.svg" alt="Medium"
+        /></a>
+        <a href="#" class="social-icon"
+          ><img src="images/icons/github.svg" alt="GitHub"
+        /></a>
+        <a href="#" class="social-icon"
+          ><img src="images/icons/angellist.svg" alt="AngelList"
+        /></a>

```

Comment:
- [ ] Please wrap your social media icons in a list, You can use `<ul>` list. When we have similar elements on the page it is a good practice to wrap them in a list and it will also make sure the code is semantic.

Reply:
Thank you 🙏, I was exactly wondering how to do it semantically. Updated.